src/App.js

```diff
@@ -1,21 +1,26 @@
 import React from 'react';
-import { createBrowserRouter, RouterProvider, Routes, Route } from 'react-router-dom';
+import { createBrowserRouter, RouterProvider, Outlet } from 'react-router-dom';
+import Navbar from './components/Navbar';
+import BookList from './components/BookList';
 
 const App = () => {
   const router = createBrowserRouter([
     {
-        path: "/",
-        element: <div>Hello world!</div>,
-      },
-      {
-        path: "/books",
-        element: <div>Hello Books!</div>,
-      },
+      path: '/',
+      element: (
+        <>
+          <Navbar />
+          <Outlet />
+        </>
+      ),
+      children: [
+        { path: '/', element: <div>Hello world!</div> },
+        { path: '/books', element: <BookList /> },
```

Comment:
- [ ] Please adhere to the project requirement, according to which there should be only two `paths` i.e. **/** for the books page as default and other **/categories** for the category page. But right now on the default page, you are using the home/default path to render `<div>Hello world!</div>` instead of the `<BookList>` component and even you forget to add the route for the `categories` page. Kindly fix the things as I requested and submit your project again.

Reply:
@skyv26 
Hi,
sure , Thanks for providing feedback

Reply:
@Xghayor Your most welcome